src/twisted/test/test_stringtransport.py

```diff
@@ -17,6 +17,7 @@
 from twisted.internet.protocol import ClientFactory, Factory
 
 
+
 class StringTransportTests(TestCase):
```

Comment:
Likewise, these tests were moved to `twisted.internet.test.test_testing`, no? So they aren't needed here anymore.

Reply:
Removed twisted.test.test_stringtransport.